lib/build.js

```diff
@@ -30,11 +30,7 @@ function findIdsOfMatchedJobs(jobs, jobNames) {
 
         // Get the actual job name if the matching job is a PR job
         if (j.isPR()) {
```

Comment:
This can be reduced to 
```
const name = j.prParentJobName || j.name;

if (jobNames.includes(name)) {
    match.push(j.id);
}

return match;
```